### PR TITLE
fix: change table name & columns

### LINKED ISSUE #10: Rename table name to category
To follow the actual convention, let's call the table `category` instead of `all_categories`

### PR DESCRIPTION
**Pull request by louisewang1**
close #10 
- change table all_categories to category
- rename levels

### CODE REVIEW

File: db-schema.sql [hunk 1]
```diff
@@ -2,59 +2,65 @@ CREATE TABLE IF NOT EXISTS category_types (
...
 );
-
-INSERT INTO category_types (id,name)
-VALUES ('3f7b79e2-7e78-4aea-b697-2b6a6ba92e91','age'),
-    ('c344bf4f-19e0-4674-b2a2-06bb5ac6e11c','discipline');
-
-CREATE TABLE IF NOT EXISTS all_categories (
+INSERT INTO category_types (name)
+VALUES ('level'),
+    ('discipline');
```
Comment (pyphilia) [inline db-schema.sql:7]: Revert these changes.

File: db-schema.sql [hunk 2]
```diff
@@ -2,59 +2,65 @@ CREATE TABLE IF NOT EXISTS category_types (
     id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
     name VARCHAR(20)
 );
-
-INSERT INTO category_types (id,name)
-VALUES ('3f7b79e2-7e78-4aea-b697-2b6a6ba92e91','age'),
-    ('c344bf4f-19e0-4674-b2a2-06bb5ac6e11c','discipline');
-
-CREATE TABLE IF NOT EXISTS all_categories (
+INSERT INTO category_types (name)
```
Comment (pyphilia) [inline db-schema.sql:5]: Actually can we also change to `category_type`?

**Review by pyphilia — CHANGES_REQUESTED** [review]
Thanks for these changes! 🍰

**Comment by louisewang1** [comment]
Thanks for your review, I have applied the changes ;)

**Pull request merged**